Orange/base.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, domain=None):
         self.domain = domain
 
     def predict(self, X):
-        if self.predict_storage == Model.predict_storage:
+        if type(self).predict_storage is Model.predict_storage:
```

Comment:
I didn't change it because of `is` but because of `type` --- the condition couldn't be `True` (I guess) since it compared bound and unbound method.

```
>>> class A:
...     def f():
...         pass
...
>>> a = A()
>>> a.f == A.f
False
>>> type(a).f == A.f
True
```